.github/workflows/backup-kubernetes-databases-inhouse.yml

```diff
@@ -114,8 +154,17 @@ jobs:
           mkdir "backup"
           
           # loop in pods which are going to be backuped
+          echo "Starting backup loop"
           for pod in ${select};
           do          
+          
+            # Check if the pod is in the comma-separated list of items to skip
+            if [[ "${{ inputs.skip }}" == *"$pod"* ]]; then
```

Comment:
Maybe I'm missing something but I don't see `skip` between workflow inputs.

Reply:
sry, forgot to remove. This was the the first approach, but then I decided to do it with `--field-selector` which si more flexible for filtering